source/guides/guides/command-line.md

```diff
@@ -351,9 +351,8 @@ DEBUG=cypress:* cypress run
 ```shell
 set DEBUG=cypress:*
 ```
-
 ```shell
-set DEBUG=cypress:*
+cypress run
```

Comment:
Command was duplicated so I changed it to reflect what the earlier Linux/Mac example was doing.